doc/develop/test/twister.rst

```diff
@@ -594,6 +605,43 @@ on those platforms.
   with the hardware map features. Boards that require other runners to flash the
   Zephyr binary are still work in progress.
 
+As mentioned in the previous section, the ``--device-serial-pty`` option, also
+can be used in multiple device testing. Like the following example of map.yaml,
+you can use ``serial_pty`` instead of ``serial_pty``, and usage is as same as
```

Comment:
what do you mean?
```
you can use ``serial_pty`` instead of ``serial_pty``, and usage is as same as
```

Reply:
It shall be "serial" for the second one, thanks so much for correcting this. Fixed.